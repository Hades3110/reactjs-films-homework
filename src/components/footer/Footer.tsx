import styles from './footer.module.scss'

const Footer = () => (
    <footer className={styles.footer}>
        <div className={styles.footer__name}>FILMS</div>
        <div className={styles.footer__info}>Copyrigth &copy 2021 FILMS. HRANT GRISHYAN</div>
    </footer>
)

export default Footer
