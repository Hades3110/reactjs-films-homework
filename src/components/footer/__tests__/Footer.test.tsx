import { create } from "react-test-renderer";
import Footer from "../Footer";

describe('Footer Component', () => {

    it('Footer snapshot test', () => {
        const component = create(<Footer />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});