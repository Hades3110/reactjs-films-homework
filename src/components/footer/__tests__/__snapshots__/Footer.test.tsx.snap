// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Footer snapshot test 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__name"
  >
    FILMS
  </div>
  <div
    className="footer__info"
  >
    Copyrigth Â© 2021 FILMS. HRANT GRISHYAN
  </div>
</footer>
`;
